### DISCUSSION TITLE
Excuse me tsyuay, can you make the penguin diner 2 game full screen?

### DISCUSSION THREAD

**Discussion by frivclassic2022** [Tsyuay/FrivReborn#43] [Q&A] [answered]
Please?
![Screenshot_20230129-000235_FlipaClip](https://user-images.githubusercontent.com/117296290/215302199-67c848c6-3b0f-4e97-be70-e98cd276172e.jpg)

**Comment by Tsyuay** [comment, marked as answer]
I can't.

**Comment by frivclassic2022** [comment]
it's all right
